management/cpu_mem_monitor/src/cpu_mem_monitor.cc

```diff
@@ -461,6 +480,56 @@ int CpuMemMonitor::CollectCPUStats() {
     cpu_state_msg_.cpus[i].loads[4] = load_cpus_[(i + 1)].total_percentage;
   }
 
+  // Go through all the node list and calculate cpu usage
+  uint64_t utime, stime;
+  double cpu_usage_node;
+  std::map<std::string, int>::iterator it;
+  std::map<std::string, uint64_t>::iterator it_load;
+  int i = -1;
+  for ( it = nodes_pid_.begin(); it != nodes_pid_.end(); it++ ) {
+    // Increment counter
+    i++;
```

Comment:
A more elegant way to do this is to put in the for loop, for (int it = -1, std::...::iterator it = ... ; ... ; i++, it++) with the comma to do multiple things